beacon-chain/state/state-native/gloas.go

```diff
@@ -0,0 +1,78 @@
+package state_native
```

Comment:
Having all Gloas logic in a `gloas.go` file is new, so far we have been splitting everything by functionality. For example `latestWithdrawalsRootVal` would go to `getters_withdrawal.go`.